pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -710,9 +710,14 @@ func (c *Cluster) failoverMon(name string) error {
 	logger.Infof("starting new mon: %+v", m)
 
 	// Scale down the failed mon to allow a new one to start
-	if err := c.updateMonDeploymentReplica(name, false); err != nil {
-		// attempt to continue with the failover even if the bad mon could not be stopped
-		logger.Warningf("failed to stop mon %q for failover. %v", name, err)
+	// If on host network, don't scale down the mon since we don't want to schedule a new
+	// mon with the same host ip. With host networking, when failing over the mon, we either
+	// require another eligible node without a mon, or we expect the down mon to come back online.
```

Comment:
Won't this make it impossible to do a mon failover on a three node cluster? If yes, then I think it might block a few scenarios: 
- When we enable submariner with RDR on provider mode.  I believe provider mode uses Host Networking.
-[ Failing over mons with hostpath to use persistent volumes.](https://github.com/rook/rook/pull/12952) 
- Failing over mons use host network.

Reply:
ignore the first part of my previous comment. RDR with provider mode does not use submariner. So no mon failover is needed.